### PR TITLE
Consul & libnetwork instructions for manual setup

### PR DESCRIPTION
**Pull request by spikecurtis**

### CODE REVIEW

File: docs/ManualClusterSetup.md [hunk 1]
```diff
@@ -4,14 +4,17 @@ This document describes requirements and best practices for setting up a cluster
 
 ## Requirements
 
-2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+2 servers (bare metal or VMs) with a modern 64-bit Linux OS, each with a unique hostname.
 
 They must have the following software installed.
  * Docker v1.8 or greater: [Experimental Docker](https://experimental.docker.com)
  * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
+ * Consul installed on one node [Consul Documentation](https://www.consul.io/docs/index.html)
```
Comment (tomdee) [inline docs/ManualClusterSetup.md]: Should we bump the etcd version up here whilst we're making changes?

File: docs/ManualClusterSetup.md [hunk 2]
```diff
@@ -4,14 +4,17 @@ This document describes requirements and best practices for setting up a cluster
 
 ## Requirements
 
-2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+2 servers (bare metal or VMs) with a modern 64-bit Linux OS, each with a unique hostname.
```
Comment (tomdee) [inline docs/ManualClusterSetup.md]: I've never liked how we say 2 servers here. We're talking about manual cluster setup, why do we require any particular number?

File: docs/ManualClusterSetup.md (hunk 1 above)
Comment (tomdee) [inline docs/ManualClusterSetup.md]: For consul, I think it's better to be consistent with how we describe our need for etcd. i.e. Consul cluster installed and accessible from each compute node.

File: docs/ManualClusterSetup.md [hunk 3]
```diff
@@ -4,14 +4,17 @@ This document describes requirements and best practices for setting up a cluster
...
 They must have the following software installed.
  * Docker v1.8 or greater: [Experimental Docker](https://experimental.docker.com)
  * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
+ * Consul installed on one node [Consul Documentation](https://www.consul.io/docs/index.html)
+   * Consul is required for Docker Networking.  etcd support is planned, but currently broken.
+   * Do not run Consul in a Docker container; it must be available when the Docker daemon starts.
  * `ipset`, `iptables`, and `ip6tables` kernel modules.
 
-## Best Practices
+## Getting Docker 1.8 Experimental Binaries
```
Comment (tomdee) [inline docs/ManualClusterSetup.md]: We should be fully supported on CoreOS now. The plugin path thing is no longer a problem. So remove the section recommending powerstrip

And the experimental binaries are never updated by docker. So current plan of record is that the docker binary is attached to the releases page. And that's the version they should use.
(Also see this https://github.com/Metaswitch/calico-docker/wiki/Current-recommended-Experimental-Docker-version-for-testing-Calico)

File: docs/ManualClusterSetup.md [hunk 4]
```diff
@@ -47,7 +52,33 @@ Get the calico binary onto each host.  You can download a specific [release](htt
 Finally, you can optionally preload the Calico Docker image.  This will make the Calico container creation more responsive the first time you run it.  Select the same version of the Calico Docker image as you selected above.  For example, to pull the latest released version
```
Comment (tomdee) [inline docs/ManualClusterSetup.md]: I think it's worth tightening up the language here around the "calico docker image" and "calico container". We should introduce the calico-node term and describe it as our agent that runs on each compute host.

**Comment by tomdee** [comment]
Looks good. A few more comments to resolve and then it might be worth me taking another quick look.

It would be good if you can squash your commits before you merge.

**Comment by spikecurtis** [comment]
Squashed.  Have another look

**Comment by spikecurtis** [comment]
Fixes #357

File: docs/ManualClusterSetup.md [hunk 5]
```diff
@@ -4,32 +4,39 @@ This document describes requirements and best practices for setting up a cluster
...
+We recommend that you download your Docker binary from [the release page](https://github.com/Metaswitch/calico-docker/releases) of the Calico version you are using, rather than experimental.docker.com, since we will have had a chance to check it functions with Calico.
 
-### Ubuntu
-The Ubuntu-managed Docker package is out of date and will not work with `calicoctl`.  We recommend you follow the instructions in the comments of the [Experimental Docker Website](https://experimental.docker.com) to get an up-to-date Docker install.
+For example, as root: 
 
-### Final checks
+    wget https://github.com/Metaswitch/calico-docker/releases/download/v0.5.2/docker
+    chmod +x docker
+    cp docker `which docker`
```
Comment (tomdee) [inline docs/ManualClusterSetup.md]: This usually requires `sudo`

You could be fancy and do:
`sudo wget -O $(which docker) https://github.com/Metaswitch/calico-docker/releases/download/v0.5.2/docker`

...which means no need for the `chmod` or the `cp`.

**Comment by tomdee** [comment]
One final comment. Otherwise LGTM

**Pull request merged**